src/client/app/containers/app.ts

```diff
@@ -78,6 +79,10 @@ export class AppComponent {
             );
     }
 
+    get isDownloadZipSupported() {
+        return Utilities.platform !== PlatformType.MAC;
```

Comment:
I'm assuming Mac & iPad are both impacted, right?
Admittedly, we aren't really testing iPad.  But the check may as well be on both, I would think...